### PR TITLE
Add daterange to slide approval

### PR DESCRIPTION
**Pull request by mirabelrice**

### CODE REVIEW

File: test/integration/send_slide_to_sign_test.rb [hunk 1]
```diff
@@ -17,6 +17,8 @@ class SendSlideToSignTest < Capybara::Rails::TestCase
     sign_out
     sign_in users(:ross)
     assert page.has_content?(/James Slide needs to be approved to .* Ross Sign/)
+    assert page.has_content?(/October 5, 2016/)
+    assert page.has_no_content?(/Ending on/)
```
Comment (tatwell) [inline test/integration/send_slide_to_sign_test.rb]: It's not clear to me what's being tested here. This is where second error message argument can be useful:

    assert page.has_no_content?(/Ending on/), "Explain assert or what went wrong here."

File: test/mailers/user_mailer_test.rb [hunk 2]
```diff
@@ -4,10 +4,15 @@ class UserMailerTest < ActionMailer::TestCase
...
     signs(:one).owners << users(:ross)
+    sign_slides(:one).slide.play_on = "2016-09-05 10:45:00"
+    sign_slides(:one).slide.stop_on = "2016-09-07 10:45:00"
     mail = UserMailer.sign_slide_request(sign_slide: sign_slides(:one), requestor: users(:james))
+ 
     assert_equal "Play Slide Request: MyString", mail.subject
     assert_equal ["loehner@chapman.edu"], mail.to
     assert_match "requesting that their slide", mail.body.encoded
+    assert_match (/September .* 5, 2016/), mail.body.encoded
+    assert_match (/September .* 7, 2016/), mail.body.encoded
```
Comment (tatwell) [inline test/mailers/user_mailer_test.rb]: Please clarify what's being asserted here, either by code comment or by using assert message arg.

File: test/integration/send_slide_to_sign_test.rb [hunk 3]
```diff
@@ -16,9 +16,10 @@ class SendSlideToSignTest < Capybara::Rails::TestCase
     assert_not slide_is_on_sign?('James Slide', signs(:one))
     sign_out
     sign_in users(:ross)
+
     assert page.has_content?(/James Slide needs to be approved to .* Ross Sign/)
-    assert page.has_content?(/October 5, 2016/)
-    assert page.has_no_content?(/Ending on/)
+    assert page.has_content?(/Starting on/) #slide has a start-date which should appear
+    assert page.has_no_content?(/Ending on/) #don't show end-date if slide doesn't have one
```
Comment (tatwell) [inline test/integration/send_slide_to_sign_test.rb]: I don't like inline comments like this (I believe they're discouraged by the Roby style guide.) Could you not get message to work? You may need to enclose assert in parens like this:

    assert(page.has_content?(/Starting on/), "Page should display start date when set for event.")
    assert(page.has_no_content?(/Ending on/), "Page should not display end date when not set for event.")

Comment (mirabelrice) [inline test/integration/send_slide_to_sign_test.rb]: I've never heard that ruby discourages inline comments in all situations. If this is something we have decided we should put it in our style guide.

**Pull request merged**